src/components/CalendarDay.tsx

```diff
@@ -0,0 +1,92 @@
+import Link from "next/link"
+import { styled } from "linaria/react"
+import { format, isToday } from "date-fns"
+
+import { Emotion, getEmotionColor, renderEmoji } from "../utils/emotions"
+
+const Container = styled.a`
+  display: grid;
+  grid-template-areas: "content";
+  justify-items: center;
+  align-items: center;
+  width: 100%;
+`
+
+const Background = styled.div<{ backgroundColor?: string; isToday?: boolean }>`
+  grid-area: content;
+  border-radius: 12px;
+  background-color: ${(props) =>
```

Comment:
These conditional css values can be tested if you export the component. See ```matches tab snapshot when active``` in https://github.com/feelingapp/feeling-app/blob/master/test/components/TabBar.test.tsx for an example